docs/ide/editorconfig-code-style-settings-reference.md

```diff
@@ -30,15 +31,15 @@ There are three supported .NET coding convention categories:
 
 - [Language code styles](#language-code-styles)
 
-   Rules pertaining to the C# or Visual Basic language. For example, you can specify rules around using `var` or explicit types when defining variables, or preferring expression-bodied members.
```

Comment:
@tiesmaster This commit seems to have introduced a number of formatting changes such as removing whitespace at the beginning of lines. Was this intentional? I would need to see how this renders before we merge it.

Reply:
Sorry, no, that was not intentionally 😬 I used VS Code for this PR, and I have [prettier](https://prettier.io/) installed for frontend work, it formats it on save (just don't know why it didn't do it for the first commit 😕).

Reverted that portion of the commit, and rebased.